charts/gsp-cluster/policies/README.md

```diff
@@ -2,11 +2,24 @@
 
 ## Running Tests
 
+All together:
+
+```
+$ opa test policies
+PASS: 21/21
+```
+
+Or, individually:
+
 ```
 $ opa test policies/digests-on-images
-PASS: 3/3
+PASS: 5/5
 ```
 ```
 $ opa test policies/restrict-special-nodes
 PASS: 11/11
 ```
+```
+$ opa test policies/isolate-tenant-istio-resources
+PASS: 5/5
```

Comment:
isn't it 6/6? and therefore all would be 22/22?

Reply:
It's a readme. It's illustrative.